easybuild/easyconfigs/l/libgtextutils/libgtextutils-0.7-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# foss-2016b modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'libgtextutils'
+version = '0.7'
+
+homepage = 'http://hannonlab.cshl.edu/fastx_toolkit/'
+description = "ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream"
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/agordon/%(name)s/releases/download/%(version)s']
+sources = [SOURCE_TAR_GZ]
+patches = ['%(name)s-%(version)s_fix-bool.patch']
+checksums = [
+    '792e0ea3c96ffe3ad65617a104b7dc50684932bc96d2adab501c952fd65c3e4a',  # libgtextutils-0.7.tar.gz
+    'bb16a4fd86c2eb12215d8780b09f0898771a73e53889a015e2351f2d737c9a00',  # libgtextutils-0.7_fix-bool.patch
+]
+
+builddependencies = [('binutils', '2.30')]
+
+sanity_check_paths = {
+    'files': ['lib/%(name)s.so', 'lib/%(name)s.a'],
```

Comment:
@SethosII Please use `SHLIB_EXT` rather than `.so`, like we do in the existing `libgtextutils` easyconfigs

Reply:
@boegel I changed it.